mods/cnc/sequences/vehicles.yaml

```diff
@@ -215,7 +215,7 @@ mlrs:
 	turret:
 		Start: 32
 		Facings: 32
-	turret-2:
+	aim:
```

Comment:
The glitch is gone, but it looks like you don't use the empty sequence at all now.